components/test_coprocessor/src/fixture.rs

```diff
@@ -94,10 +94,10 @@ pub fn init_data_with_details<E: Engine>(
     store.begin();
     for &(id, name, count) in vals {
         store
-            .insert_into(&tbl.table)
-            .set(&tbl.id, Datum::I64(id))
-            .set(&tbl.name, name.map(|s| s.as_bytes()).into())
-            .set(&tbl.count, Datum::I64(count))
+            .insert_into(tbl.as_table())
```

Comment:
I think `insert_into(&tbl)` can work?

Reply:
Nice. I don't know this will work. I think I can remove `as_table` now.